### ISSUE TITLE
Support example table for scenario outline

### ISSUE THREAD

**Issue by Xzirez** [olexale/bdd_widget_test#17]
Hi, it would be great if the plugin could support a scenario outline table. Currently, we want to use the plugin in combination with X-Ray. Our QA writes a value table like in the following example:

```
Scenario: Users is able to create a New Home (only required fields)
    Given The user is registered
    And The user is logged
    And The USER_MENU page is opened
    When The user tapped the +HOME button
    And The SAVE button is inactive
    And The user filled in the required text fields: <Type of home>, <Address>, <Adults>
        | Type of home | Address                    | Adults |
        | House        | Nedre Storgate, 42 Drammen | 2      |
    Then The SAVE button becomes active
    And The user tapped the SAVE button
    And The user is redirected to the MY_HOME page
    And The Home is created
    And The text fields are contain entered data: <Type of home>, <Address>, <Adults>
        | Type of home | Address                    | Adults |
        | House        | Nedre Storgate, 42 Drammen | 2      |

```
Without this functionality, there would be some limitations in the BDD process. Currently, we are trying to make some changes to the plugin to accommodate such tables and would be happy to help with any future improvements to the plugin. If there are other suggestions for similar improvements it would also be great. Thank you.

**Comment by olexale** [comment]
Hi @Xzirez,

[Scenario Outline](https://cucumber.io/docs/gherkin/reference/#scenario-outline) has a bit different syntax (which I think suits your sample even better as you will not get a data duplication) but sure - the plugin needs to support `Data Tables` as well. It would be awesome if you can share your changes so the plugin might become better.

Thanks in advance!

Kind regards,
Oleksandr

**Comment by Xzirez** [comment]
Hi @olexale 

Thank you for the reply. Our team member Suren made a fork that we are currently using. It's still being worked on and we will make a PR with some of the changes that would be more universally useful. The commit history should be clean enough. Here is the fork https://github.com/sosuren/bdd_widget_test

BR :)
Vetle

**Comment by olexale** [comment]
Hi @Xzirez,

I just published version 1.2.0 with "Scenario Outline" support. This is not exactly what you have mentioned in your example but you may still find it useful. Thanks!

Kind regards,
Oleksandr

**Comment by olexale** [comment]
Hi all,

`DataTables`-like syntax is implemented in `1.2.1`. Your sample should work now. Feel free to re-open the ticket if you have issues with it or if you have any propositions on improvements.

Best,
Oleksandr
